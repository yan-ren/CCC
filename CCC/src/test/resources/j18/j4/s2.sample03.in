3
3 7 9
2 5 6
1 3 4
