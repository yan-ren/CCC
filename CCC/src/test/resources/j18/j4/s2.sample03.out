1 2 3
3 5 7
4 6 9
